src/components/inputs/Dates/Calendar/Calendar.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactElement } from 'react';
+import '@testing-library/jest-dom';
+import {
+  render,
+  RenderOptions,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { ThemeProvider } from 'styled-components';
+import { themes } from '../../../../themes';
+import { Calendar } from './Calendar';
+
+const Provider = ({ children }) => {
+  return (
+    <ThemeProvider theme={themes['light']}>{children}</ThemeProvider>
+  );
+};
+
+const renderWithThemeProvider = (
+  ui: ReactElement,
+  options?: Omit<RenderOptions, 'wrapper'>
+) => render(ui, { wrapper: Provider, ...options });
+
+describe('Calendar', () => {
+  it('Renders Calendar', () => {
+    renderWithThemeProvider(<Calendar data-testid="calendar" />);
+    const calendar = screen.getByTestId('calendar');
+    expect(calendar).toBeInTheDocument();
+  });
+
+  it('Renders with initial month', () => {
+    const date = new Date(2022, 8, 26);
+
+    renderWithThemeProvider(
+      <Calendar data-testid="calendar" initialMonth={date} />
+    );
+
+    const month = screen.getByRole('heading').innerHTML;
+    expect(month).toBe('September 2022');
```

Comment:
I think we should be able to use the `toHaveTextContent` matcher
```suggestion
    const month = screen.getByRole('heading');
    expect(month).toHaveTextContent('September 2022');
```

Reply:
Ah! thanks!!